Document `omit_row` noise type (#143)
ihmeuw/pseudopeople: docs/source/noise/row_noise.rst

Row-based Noise
===============

> Row-based noise operates on one row of data at a time, for example by
> introducing errors in cells within certain rows, or by omitting or duplicating

Reviewer: ```suggestion
Row-based noise operates on an entire row of data at a time, for example by omitting or duplicating
```

I see where you are going with this, but I think the "for example" already implies enough flexibility.

I'm not sure "introducing errors in cells within certain rows" gives me _that_ much more information beyond what "operates on one row of data at a time" already tells me.

Saying "an entire row" instead of "one row" makes it sound like a lot (rather than a little), which is what it is relative to column-based noise.

Author: I changed "an entire" back to "one" because I already say "entire" later in the sentence.
